packages/features/ee/teams/pages/team-appearance-view.tsx

```diff
@@ -46,7 +46,7 @@ const ProfileView = () => {
 
   return (
     <>
-      <Meta title="Booking Appearance" description="Manage settings for your team's booking appearance" />
+      <Meta title={t("booking_appearance")} description={t("appearance_team_description")} />
```

Comment:
@OMaxOe can you confirm that while loading this page, there is no flicker of untranslated window title.

Reply:
For context, https://github.com/calcom/cal.com/pull/4354/files

Reply:
@hariombalhara I haven´t noticed any flicker of untranslated text while loading

Reply:
There is a flicker that's visible in https://www.loom.com/share/bd9aa0051e4f4b979618a71cbfc6370d

Reply:
The way to fix it is to do translations serverside, either during SSG or SSR. See this as an example https://github.com/calcom/cal.com/pull/5098/files

Reply:
I think it's because you are using ssgInit inside getServerSideProps. There is `ssrInit` for that. `ssgInit` is for `getStaticProps`

Reply:
@hariombalhara ok I have fixed the flickering on all pages. On some pages the flickering occurred because no title was set while the page is in loading state. Please test it again.